pkg/get/get.go

```diff
@@ -176,6 +176,9 @@ func getByDownloadTemplate(tool Tool, os, arch, version string) (string, error)
 		"OS":      os,
 		"Arch":    arch,
 		"Version": version,
+		"Owner":   tool.Owner,
+		"Repo":    tool.Repo,
+		"Name":    tool.Name,
```

Comment:
Rather than adding these to each template run, add the values inside your tool's template to match the other tools

Reply:
All set here